crates/shared/src/subgraph.rs

```diff
@@ -71,31 +71,32 @@ impl SubgraphClient {
 
     /// Performs the specified GraphQL query on the current subgraph.
     /// This function should be called for queries that return very long(paginated) result.
-    pub async fn paginated_query<T>(&self, block_number: u64, query: &str) -> Result<Vec<T>>
+    pub async fn paginated_query<T>(
+        &self,
+        query: &str,
+        variables: Option<Map<String, Value>>,
```

Comment:
```suggestion
        variables: Map<String, Value>>,
```
The map can already be empty to indicate no parameters so no need for Option.

Reply:
Yes, seems a bit simpler without `Option`. Fixed.